doc/sphinx/source/recipes/recipe_pv_capacity_factor.rst

```diff
@@ -0,0 +1,76 @@
+.. _recipes_pv_capacity_factor:
+
+Capacity factor for solar photovoltaic (PV) systems
+===================================================
+
+Overview
+--------
+
+This diagnostic computes the PV factor, a measure of the fraction of the 
+maximum possible energy produced per PV grid cell. It uses the daily incoming 
+surface solar radiation and the surface temperature with a method described
+in `Bett and Thornton (2016)`_. The user can select temporal
+range, season, and region of interest.
+
+
+.. _`Bett and Thornton (2016)`: https://doi.org/10.1016/j.renene.2015.10.006
+
+
+Available recipes and diagnostics
+---------------------------------
+
+Recipes are stored in recipes/
+
+* recipe_pv_capacity_factor.yml
+
+Diagnostics are stored in diag_scripts/pv_capacityfactor/
+
+* pv_capacity_factor.R: prepares data and plots results.
+* PV_CF.R: calculates the daily capacity factor.
+
+
+User settings
+-------------
+
+User setting files are stored in recipes/
+
+#. recipe_capacity_factor.yml
+
+   *Required settings for script*
+
+   * season: String to include shortcut for season in plot title and name (e.g. "djf").
+     It will be converted to upper case. This season should be the one set in the preprocessor,
+     since it is only used as a string and does not affect the data in the diagnostic.
+     In the default recipe this is solved through a node anchor.
+   
+   *Optional settings for script*
+   
+   * maxval_colorbar: Optional upper limit for the colorbar.
+
+Variables
+---------
+
+* tas (atmos, daily, longitude, latitude, time)
+* rsds (atmos, daily, longitude, latitude, time)
+
+
+Observations and reformat scripts
+---------------------------------
+
+* ERA-Interim
+
+References
+----------
+
+* Bett, P. E. and Thornton, H. E.: The climatological relationships between wind and solar energy supply in Britain, Renew. Energ., 87, 96–110, https://doi.org/10.1016/j.renene.2015.10.006, 2016.
+
+
+Example plots
+-------------
+
+.. _fig_pv_capfactor1:
+.. figure::  /recipes/figures/pv_capacity_factor/capacity_factor_IPSL-CM5A-MR_1980-2005_DJF.png
```

Comment:
The recipe has ERA-Interim and several CMIP5 Models as input, among them IPSL-CM5A-MR. I don't remember, why exactly this model was chosen, but it is one used in the example recipe and in the paper. As ESMValTool is primarily made for the evaluation of models I'd like to keep a model there or use a model and ERA-Interim?

ERA-Interim is mentioned in the documentation, because there usually the only observational and reanalysis data sets are named because it is assumed that recipes usually run with any CMIP5 or CMIP6 model. But in contrast to the models observations often do not provide more than a few input variables (some more for reanalysis). ERA-Interim is one of the few of these data sets, which provides both rsds and tas as variables.

Reply:
Thanks for explaining. I think keeping IPSL is fine.